.pylintrc

```diff
@@ -14,7 +14,8 @@ ignore=CVS,settings.py
 
 # Add files or directories matching the regex patterns to the blacklist. The
 # regex matches against base names, not paths.
-ignore-patterns=
+# ^[0-9]{4}_.*\.py$ : ignore migration files
+ignore-patterns=^[0-9]{4}_.*\.py$
```

Comment:
Just curious what were the linting issue in the migration file that you blacklisted them ?

Reply:
When you do data migrations you have to 'generate' a representation of the model at the time of the migration (might be different to the current version in the code). This Model variable is typically uppercase (since the model itself is uppercase), and pylint doesn't like this.